src/views/medewerkers/MedewerkerList.vue

```diff
@@ -0,0 +1,194 @@
+					:name="getName(medewerker)"
+					:active="medewerkerStore.medewerkerItem?.id === medewerker?.id"
+					:force-display-actions="true"
+					:details="_.upperFirst(medewerker.type)"
+					@click="medewerkerStore.setMedewerkerItem(medewerker)">
+					<template #icon>
+						<AccountOutline :class="medewerkerStore.medewerkerItem === medewerker.id && 'selectedZaakIcon'"
+							disable-menu
+							:size="44" />
+					</template>
+					<template #subname>
+						{{ getSubname(medewerker) }}
+					</template>
+					<template #actions>
+						<NcActionButton @click="medewerkerStore.setMedewerkerItem(medewerker); navigationStore.setModal('editMedewerker')">
+							<template #icon>
+								<Pencil :size="20" />
+							</template>
+							Bewerken
+						</NcActionButton>
+					</template>
+				</NcListItem>
+			</div>
+		</ul>
+
+		<div v-if="!medewerkerStore.medewerkersList?.length && !loading">
+			Geen medewerkers gedefinieerd.
+		</div>
+
+		<NcLoadingIcon v-if="loading"
+			class="loadingIcon"
+			:size="64"
+			appearance="dark"
+			name="Medewerkers aan het laden" />
+	</NcAppContentList>
+</template>
+<script>
+// Components
+import { NcListItem, NcActionButton, NcAppContentList, NcTextField, NcLoadingIcon, NcActions } from '@nextcloud/vue'
+import _ from 'lodash'
+
+// Icons
+import Magnify from 'vue-material-design-icons/Magnify.vue'
+import AccountOutline from 'vue-material-design-icons/AccountOutline.vue'
+import Refresh from 'vue-material-design-icons/Refresh.vue'
+import Plus from 'vue-material-design-icons/Plus.vue'
+import Pencil from 'vue-material-design-icons/Pencil.vue'
+
+export default {
+	name: 'MedewerkerList',
+	components: {
+		// Components
+		NcListItem,
+		NcActionButton,
+		NcAppContentList,
+		NcTextField,
+		NcLoadingIcon,
+		// Icons
+		AccountOutline,
+		Magnify,
+		Pencil,
+	},
+	data() {
+		return {
+			search: '',
+			loading: true,
+			medewerkersList: [],
+		}
+	},
+	mounted() {
+		medewerkerStore.refreshMedewerkersList().then(() => {
+			this.loading = false
+		})
+	},
+	methods: {
+		fullName(medewerker) {
+			let name = medewerker.achternaam
+			if (medewerker.tussenvoegsel) {
+				name = `${medewerker.tussenvoegsel} ${name}`
+			}
+			if (medewerker.voornaam) {
+				name = `${name}, ${medewerker.voornaam}`
+			}
+			return name
+		},
+		getName(medewerker) {
+			return medewerker?.voornaam ?? 'onbekend'
+
+		},
+		getSubname(medewerker) {
+			return medewerker?.tussenvoegsel ? `${medewerker.tussenvoegsel} ${medewerker.achternaam}` : medewerker?.achternaam ? `${medewerker.achternaam}` : 'onbekend'
+
+		},
+		deleteKlant() {
+			fetch(
+				`/index.php/apps/zaakafhandelapp/api/medewerkers/${medewerkerStore.medewerkerItem.id}`,
+				{
+					method: 'DELETE',
+				},
+			)
+				.then((response) => {
+					response.json().then((data) => {
+						this.medewerkersList = data
+					})
+					this.loading = false
+				})
+				.catch((err) => {
+					console.error(err)
+					this.loading = false
+				})
```

Comment:
dit zou via de klant store moeten gaan.
Nou weet ik niet uit mijn hoofd of de klant store up-to-date is, zo niet laat maar zitten, het kan wel meegenomen worden met een refactor.